cmd/commands/server/options.go

```diff
@@ -125,13 +122,14 @@ func ParseArguments(args []string) (options CommandOptions, err error) {
 	flags.StringVar(
 		&options.DiscoveryAPIAddress,
 		"discovery-address",
-		cmd.MysteriumAPIURL,
+		"",
```

Comment:
Would be nice to print current defaults as was before

Reply:
There are no defaults anymore

Reply:
There are, DEVNET is default, I want to show it clearly

Reply:
Testnet is default as defined in network.go there are now default override values.

Reply:
We already started to use terminology Testnet/Devnet or TESTNET/DEVNET in communication, event You in previous sentence

Reply:
Ok I will stick to "Testnet" "Devnet" naming